angelssmelting/prototypes/technology/smelting-cobalt.lua

```diff
@@ -9,8 +9,8 @@ data:extend(
       icon_size = 256, icon_mipmaps = 4,
       upgrade = true,
       prerequisites = {
-        "ore-floatation",
-        "angels-metallurgy-2",
+        "ore-leaching",
+        "angels-metallurgy-3",
         "angels-coal-processing"
```

Comment:
Since cobalt smelting went up a tier, tungsten smelting 1 depends now on purple science, because it depends on cobalt powder.
![image](https://user-images.githubusercontent.com/26593477/122793601-af7f3980-d2bb-11eb-9edf-3a3babcc3573.png)
It is safe to move cobalt powder to cobalt smelting 1 and adapt the tungsten prerequisite from cobalt 2 to cobalt 1.

Reply:
However that should only be done when not in science mode (in science mode however, tungsten 1 needs to shift to blue science)
![image](https://user-images.githubusercontent.com/26593477/122796605-de4adf00-d2be-11eb-8b38-1af5e894b7d0.png)

Reply:
fixed in https://github.com/Arch666Angel/mods/pull/608/commits/7248ef7f0ea9cec61cc8e24961eb0fec40660863
![image](https://user-images.githubusercontent.com/26593477/122811447-c8461a00-d2d0-11eb-8b43-6899b4c1034c.png)